features/step_definitions/finance/invoicing_steps.rb

```diff
@@ -87,17 +87,17 @@
 end
 
 Then /^the buyer should have (\d+) invoices?$/ do |number|
-  set_current_domain @provider.external_domain
-  try_buyer_login_internal(@buyer.admins.first.username, "supersecret")
+  step "the buyer logs in"
+
   visit admin_account_invoices_path
 
   assert_selector(:css, 'tr.invoice', count: number.to_i)
 end
 
 Then /^the buyer should have following line items for "([^"]*)"(?: in the (\d)(?:nd|st|rd|th))? invoice:$/ do |date, order, items|
   ActiveSupport::Deprecation.warn '[Cucumber] Deprecated! Assert table instead.'
-  set_current_domain @provider.external_domain
-  try_buyer_login_internal(@buyer.admins.first.username, "supersecret")
+  step "the buyer logs in"
```

Comment:
Replacing with step to reduce duplication. It needed user assertion to be added.

Reply:
`step` is being deprecated by Cucumber if I remember correctly. So we may need to probably have a library of methods that would be used in multiple steps. But not a request for a change now, just saying.